src/axom/inlet/Table.hpp

```diff
@@ -730,10 +875,116 @@ class Table : public SchemaCreator, public std::enable_shared_from_this<Table>
+                                          const std::string& lookupPath);
+
+template <>
+void Table::addPrimitiveArrayHelper<int>(Table& table,
+                                         const std::string& lookupPath);
+
+template <>
+void Table::addPrimitiveArrayHelper<double>(Table& table,
+                                            const std::string& lookupPath);
+
+template <>
+void Table::addPrimitiveArrayHelper<std::string>(Table& table,
+                                                 const std::string& lookupPath);
+
+/*!
+   *****************************************************************************
+   * \brief A wrapper class that enables constraints on groups of Tables
+   *****************************************************************************
+  */
+class AggregateTable : public std::enable_shared_from_this<AggregateTable>,
+                       public Verifiable
+{
+public:
+  AggregateTable(std::vector<std::shared_ptr<Verifiable>>&& tables)
+    : m_tables(std::move(tables))
+  { }
+
+  /*!
+   *****************************************************************************
+   * \brief This will be called by Inlet::verify to verify the contents of this
+   *  Table and all child Tables/Fields of this Table.
+   *****************************************************************************
+   */
+  bool verify();
+
+  /*!
+   *****************************************************************************
+   * \brief Set the required status of this Table.
+   *
+   * Set whether this Table is required, or not, to be in the input file.
+   * The default behavior is to not be required.
+   *
+   * \param [in] isRequired Boolean value of whether Table is required
+   *
+   * \return Shared pointer to this instance of Table
+   *****************************************************************************
+   */
+  std::shared_ptr<Verifiable> required(bool isRequired);
+
+  /*!
+   *****************************************************************************
+   * \brief Return the required status of this Table.
+   *
+   * Return that this Table is required, or not, to be in the input file.
+   * The default behavior is to not be required.
+   *
+   * \return Boolean value of whether this Table is required
+   *****************************************************************************
+   */
+  bool required();
+
+  /*!
+   *****************************************************************************
+   * \brief Registers the function object that will verify this Table's contents
+   * during the verification stage.
+   * 
+   * \param [in] The function object that will be called by Table::verify().
+   *****************************************************************************
+  */
+  std::shared_ptr<Verifiable> registerVerifier(std::function<bool(Proxy&)> lambda);
+
+private:
+  std::vector<std::shared_ptr<Verifiable>> m_tables;
 };
 
 }  // end namespace inlet
-}  // end namespace axom
+}  // namespace axom
```

Comment:
why did the `end` get removed ?

I think clang-format should add it back... unless there is an issue with the namespaces?